states/idle/gamen.mod.py

```diff
@@ -48,7 +47,7 @@ def run(self):
         font10 = ImageFont.truetype(font_path, size=10)
 
         while not self.killed:
-            game_status = self.last_status = self.get_response()
+            game_status = self.get_response()
```

Comment:
Hiermee wordt dus elke seconde je site gehit, is dat echt nodig?

Reply:
`get_response` zorgt zelf dat-ie slechts eens per 15 seconden een request stuurt (lines 27-29).

Reply:
dat is in check(), niet in get_response()

Reply:
Fixed